frontend/src/libs/components/header/header.tsx

```diff
@@ -67,12 +52,15 @@ const Header: React.FC<Properties> = ({ user }) => {
               WRITORIUM
             </Link>
 
-            <div className={styles.avatarWrapper} ref={avatarReference}>
+            <button
+              className={styles.avatarButton}
+              onClick={handleClickOnAvatar}
+            >
```

Comment:
Maybe it should be replaced with the custom Button component

Reply:
button here is just a wrapper for Avatar component. I can't use a custom Button instead because it doesn't allow children